src/server/scripts/Spells/spell_evoker.cpp

```diff
@@ -139,6 +143,61 @@ class spell_evo_blessing_of_the_bronze : public SpellScript
     }
 };
 
+std::vector<uint32> empoweredSpells{ SPELL_EVOKER_ETERNITY_SURGE, SPELL_EVOKER_FIRE_BREATH };
+
+// Called by 357212 - Pyre (Red) and 356995 - Disintegrate (Blue)
+class spell_evo_causality : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_EVOKER_CAUSALITY });
```

Comment:
if SPELL_EVOKER_CAUSALITY exists, EFFECT_0 exists, but EFFECT_1 isn't guaranteed to.